lec09-Randomization-tests.Rmd

```diff
@@ -162,7 +170,8 @@ mean_males <- mean(df_males$width)
 mean_females <- mean(df_females$width)
 
 diff_means_obs <- mean_males - mean_females
-cat("The observed difference in mean body width between male and female Hercules beetles is", diff_means_obs)
+cat("The observed difference in mean body width between 
+    male and female Hercules beetles is", diff_means_obs)
```

Comment:
Any reason you use `cat` verse using inline R code? Like so:

```
The observed difference in mean body width between male 
and female Hercules beetles is `r diff_means_obs`.
```

This way you get the nicely rendered text, unlike `cat` which prints as a text code chunk.